pebble_tool/commands/emucontrol.py

```diff
@@ -253,3 +253,22 @@ def add_parser(cls, parser):
         parser.add_argument('--direction', choices=['x+', 'x-', 'y+', 'y-', 'z+', 'z-'], default='x+',
                             help="Set the direction of the accel tap in the emulator")
         return parser
+
+
+class EmuTimeFormatCommand(PebbleCommand):
+    """Sets the emulated time format (12h or 24h)."""
+    command = 'emu-time-format'
+    valid_connections = {'qemu', 'emulator'}
+
+    def __call__(self, args):
+        super(EmuTimeFormatCommand, self).__call__(args)
+        is24Hour = args.format == "24h"
+        time_format_input = QemuTimeFormat(is24Hour=is24Hour)
+        send_data_to_qemu(self.pebble.transport, time_format_input)
+
+    @classmethod
+    def add_parser(cls, parser):
+        parser = super(EmuTimeFormatCommand, cls).add_parser(parser)
+        parser.add_argument('--format', choices=['12h', '24h'], default='24h',
```

Comment:
It doesn't make sense for this to have a default; it should bail if used without specifying.